diamm/views/website/aboutpages.py

```diff
@@ -7,17 +7,17 @@
 from rest_framework import status
 
 
-
 class AboutPagesDetail(views.APIView):
```

Comment:
If you write this view as a subclass of `generic.RetriveAPIView`, then you can set the queryset object. This will automatically include the appropriate behaviour for the `get` method, so you won't need to override it.

You can instead override the `get_queryset` method to get the queryset filtered by the current URL. So:

```python
def get_queryset(self):
    queryset = AboutPages.objects.filter(url=self.request.path)
    return queryset
```

(or something like that -- I'm not sure if that's how to access the original request object...)